tests/Integration/Resources/EcommerceBridgeTest.php

```diff
@@ -15,80 +15,278 @@
+        
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertContains(static::STORE_ID, $response->toArray());
+    }
+    
+    /** @test */
+    public function sendSyncMessages()
+    {
+        $response = $this->resource->sendSyncMessages(
+                static::STORE_ID,
+                'CONTACT',
+                [
+                    [
+                        'action' => 'UPSERT',
+                        'changedAt' => $this->getTimestamp(),
+                        'externalObjectId' => '12345',
+                        'properties' => [
+                            'firstname' => 'Jeff' . uniqid(),
+                            'lastname' => 'David',
+                            'customer_email' => 'test@example.com'
+                        ]
+                    ]
+                ]
+            );
+        
         $this->assertEquals(204, $response->getStatusCode());
+        sleep(1);
     }
-
-    public function testUpsertSettings()
+    
+    /** @test */
+    public function getAllSyncErrorsAccount()
     {
-        $settings = [
-            'enabled' => false,
-            'importOnInstall' => false,
-            'productSyncSettings' => [
-                'properties' => [
-                    ['propertyName' => 'test_name', 'dataType' => 'STRING', 'targetHubspotProperty' => 'name'],
-                ],
-            ],
-            'dealSyncSettings' => [
-                'properties' => [],
-            ],
-            'lineItemSyncSettings' => [
-                'properties' => [],
-            ],
-            'contactSyncSettings' => [
-                'properties' => [],
-            ],
-        ];
-
-        $response = $this->ecommerceBridge->upsertSettings($settings);
+        $response = $this->resource->getAllSyncErrorsAccount();
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertArrayHasKey('results', $response->toArray());
+    }
+    
+    /** @test */
+    public function checkSyncStatus()
+    {
+        $response = $this->resource->checkSyncStatus(
+                static::STORE_ID,
+                'CONTACT',
+                '12345'
+            );
 
         $this->assertEquals(200, $response->getStatusCode());
-        $this->assertEquals($settings, $response->toArray());
+    }
+    
+    /** @test */
+    public function importData()
+    {
+        $this->markTestSkipped();
+        $response = $this->resource->importData(
+                $this->getTimestamp(),
+                1,
+                [
+                    [
+                        'externalObjectId' => 'deal1',
+                        'properties' => [
+                            'firstname' => 'ever',
+                            'phone_number' => '+375441234567',
+                            'familyname' => 'greatest',
+                            'customer_email' => 'testingapis@hubspot.com',
+//                            'firstname' => 'Jeff' . uniqid(),
```

Comment:
Please remove comments

Reply:
Can't remove it 
Waiting for response from Zack